bin/modules/fetchToken.js

```diff
@@ -25,27 +25,52 @@ module.exports = (rememberedToken) => {
       if (!exists) {
         rej(err("No token found!"));
       } else {
+        const UNLOCK_EXISTING_TOKEN = "Unlock existing token";
```

Comment:
I know you may like you're underscores, but the rest of the code in this repo tends to follow camelCase instead if you don't mind changing that.

Also (more of a suggestion) you could store these strings in an object since they are both related? Something like:
```javascript
const tokenActions =  {
   unlock: "Unlock existing token",
   create: "Create new token"
}
```

Thanks :+1: